geode-docs/developing/expiration/configuring_data_expiration.html.md.erb

```diff
@@ -21,14 +21,14 @@ limitations under the License.
 
 Configure the type of expiration and the expiration action to use.
 
-<a id="configuring_data_expiration__section_ADB8302125624E01A808EA5E4FF79A5C"></a>
+-   Set the region's `statistics-enabled` attribute to `true`.
 
--   Set the region's `statistics-enabled` attribute to true.
-
-    The statistics used for expiration are available directly to the application through the `CacheStatistics` object returned by the `Region` and `Region.Entry` `getStatistics` methods. The `CacheStatistics` object also provides a method for resetting the statistics counters.
+    **Note:** Region level statistics are required to utilize expiration actions; cluster and server level statistics are not verbose enough to allow expiration actions in a region.
```

Comment:
I think this note is a bit confusing. Many of our cluster and server level statistics are at the region level. In this context (i.e. enabling region stats for expiration) I would just include a note that enabling statistics on a region is not the same thing as enabling statistics on the cluster.